apps/spruce/src/pages/adminSettings/tabs/AnnouncementsTab/AnnouncementTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const AnnouncementTab = () => (
+  <div className="flex flex-col gap-4">
+    <h1 className="text-xl font-bold">Announcements</h1>
```

Comment:
I know this is just a placeholder but these classNames aren't supported by our code